src/plugins/english/novelfull.ts

```diff
@@ -119,7 +119,10 @@ class NovelFull implements Plugin.PluginBase {
     loadedCheerio('#chapter-content div.ads').remove();
     const chapterText = loadedCheerio('#chapter-content').html() || '';
 
-    return chapterText;
+    return chapterText.replace(
+      /If you find any errors \(Ads popup, ads redirect, broken links, non-standard content, etc\.\.\), Please let us know \S* report chapter \S* so we can fix it as soon as possible\./,
```

Comment:
Is this text not part of any element that could be removed instead of matching and replacing it entirely?

Reply:
You could check the HTML of the mobile version of the site and remove the element if it exists. That would be a better solution.

Reply:
Not possible. I lack the skills.